### PR TITLE
Move Rocksmq page info log to Rokcsmq monitor

### PR DESCRIPTION
**Pull request by aoiasd**
related:https://github.com/milvus-io/milvus/issues/20021
Signed-off-by: aoiasd <zhicheng.yue@zilliz.com>

### CODE REVIEW

**Comment by aoiasd** [comment]
rerun ut

File: internal/mq/mqimpl/rocksmq/server/rocksmq_impl_test.go [hunk 1]
```diff
@@ -1057,3 +1057,43 @@ func TestRocksmq_updateAckedInfoErr(t *testing.T) {
...
+		GroupName: groupName,
+	}
+
+	_ = rmq.DestroyConsumerGroup(topicName, groupName)
+	rmq.CreateConsumerGroup(topicName, groupName)
+	rmq.RegisterConsumer(consumer)
+
+	rmq.kv = &rocksdbkv.RocksdbKV{}
+
+	rmq.Info()
```
Comment (jiaoew1991) [inline internal/mq/mqimpl/rocksmq/server/rocksmq_impl_test.go:1098]: need assertion

File: internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go [hunk 2]
```diff
@@ -278,22 +278,36 @@ func (rmq *rocksmq) Close() {
...
+		}
+
+		msgSizeKey := MessageSizeTitle + topic
+		msgSizeVal, err := rmq.kv.Load(msgSizeKey)
+		if err != nil {
+			log.Error("Rocksmq get last page size failed", zap.String("topic", topic))
+			return true
+		}
+
+		log.Info("Rocksmq Info", zap.String("topic", topic), zap.Int("consumer num", len(consumerList)), zap.Any("group names", consumersGroupName), zap.Any("group positions", consumersPosition), zap.Int("page sum", len(pages)), zap.String("last page size", msgSizeVal))
```
Comment (jiaoew1991) [inline internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go:310]: each field in separated lines, and do not use zap.Any

**Pull request merged**